state/migration_export.go

```diff
@@ -856,24 +856,24 @@ func (e *exporter) addApplication(ctx addApplicationContext) error {
 			endpoints[k] = ep.Name
 		}
 
-		userMap, err := e.st.GetOfferUsers(offer.OfferUUID)
-		if err != nil {
-			return errors.Annotatef(err, "ACL for offer %s", offer.OfferName)
-		}
-
-		var acl map[string]string
-		if len(userMap) != 0 {
-			acl = make(map[string]string, len(userMap))
-			for user, access := range userMap {
-				acl[user] = accessToString(access)
-			}
-		}
+		// TODO(aflynn): reimplement migration of offer users in domain.
```

Comment:
If there is a card to do this in the offer domain epic, no need to comment out with a TODO.

Reply:
There is a [task now](https://warthogs.atlassian.net/browse/JUJU-6577), todo removed